app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -34,14 +34,13 @@ class DisabilityCompensationJob
 
     def perform(form526_submission_id, full_name)
       form526_submission = Form526Submission.find(form526_submission_id)
+      lh_data = LighthouseVetsHealth.new(form526_submission)
```

Comment:
I'd be more comfortable calling this "lighthouse_data" or simply "data". "lh_data" is a little confusing to read since you have to mentally keep track of abbreviations at that point.